src/blog/v8-release-83.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'V8 release v8.3'
+author: 'Victor Gomes, safely working from home'
+avatars:
+ - 'victor-gomes'
+data: 2020-04-30
+tags:
+ - release
+description: 'V8 v8.3 features faster ArrayBuffers, bigger Wasm memories and deprecated APIs.'
+---
+
+Every six weeks, we create a new branch of V8 as part of our [release process](https://v8.dev/docs/release-process). Each version is branched from V8's Git master immediately before a Chrome Beta milestone. Today we're pleased to announce our newest branch, [V8 version 8.3](https://chromium.googlesource.com/v8/v8.git/+log/branch-heads/8.3), which is in beta until its release in coordination with Chrome 83 Stable in several weeks. V8 v8.3 is filled with all sorts of developer-facing goodies. This post provides a preview of some of the highlights in anticipation of the release.
+
+## Performance
+
+### Faster ArrayBuffer tracking in the garbage collector
```

Comment:
Nit: here and around wrap `ArrayBuffer` in backticks.

Reply:
Done! 🙂